src/components/letter-bag/LetterBag.js

```diff
@@ -0,0 +1,36 @@
+import { createInitialLetterBag } from '../../utils';
+import Letter from '../letter/Letter';
+
+const LetterBag = () => {
+  const letterBag = createInitialLetterBag();
+
+  return (
+    <div className="text-center">
+      <h2>Letter Bag</h2>{' '}
+      {Object.entries(letterBag || {}).map(([letter, count]) => (
+        <Letter key={`letter-${letter}`} letter={letter[0]} />
```

Comment:
instead of passing props to `Letter`, let each `Letter` figure out how to render